platform/migration/platform-migration/src/main/java/org/codice/ddf/configuration/admin/ConfigurationAdminMigratable.java

```diff
@@ -155,8 +160,12 @@ private boolean isAcceptedConfigurationAdminEntry(ImportMigrationConfigurationAd
     if (ACCEPTED_ENTRY_PIDS.contains(entry.getPid())) {
       return true;
     }
-    if (ACCEPTED_ENTRY_FACTORY_PIDS.contains(entry.getFactoryPid())) {
-      return true;
+    if (entry.getFactoryPid() != null) {
+      if (ACCEPTED_ENTRY_FACTORY_PIDS.contains(entry.getFactoryPid())
+          || StringUtils.containsIgnoreCase(entry.getFactoryPid(), "source")
+          || StringUtils.containsIgnoreCase(entry.getFactoryPid(), "service")) {
```

Comment:
✏️ Could we put these 2 new conditions in a private method like `private boolean isASource(String factoryPid)`? That way, it's more clear what these conditions are really looking for.